src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementTranslator.java

```diff
@@ -120,6 +122,17 @@ public  NewsArticle translateNewsArticle(NewsArticleId kuntaApiNewsArticleId, Po
     
     return newsArticle;
   }
+
+  public fi.otavanopisto.kuntaapi.server.rest.model.Announcement translateAnnouncement(AnnouncementId kuntaApiAnnouncementId, Announcement managementAnnouncement) {
+    fi.otavanopisto.kuntaapi.server.rest.model.Announcement result = new fi.otavanopisto.kuntaapi.server.rest.model.Announcement();
+      
+    result.setContents(managementAnnouncement.getContent().getRendered());
+    result.setId(kuntaApiAnnouncementId.getId());
+    result.setPublished(toOffsetDateTime(managementAnnouncement.getDate()));
+    result.setTitle(managementAnnouncement.getTitle().getRendered());
+    
+    return result;
+  }
   
   private AttachmentId getImageAttachmentId(OrganizationId organizationId, Integer id) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "getImageAttachmentId" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)